tests/playwright-test/max-failures.spec.ts

```diff
@@ -79,3 +79,37 @@ test('max-failures should work with retries', async ({ runInlineTest }) => {
   expect(result.failed).toBe(1);
   expect(result.output.split('\n').filter(l => l.includes('Received:')).length).toBe(2);
 });
+
+test('max-failures should stop workers', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.spec.js': `
+      const { test } = pwt;
+      test('passed', async () => {
+        await new Promise(f => setTimeout(f, 1000));
+      });
+      test('failed', async () => {
+        test.expect(1).toBe(2);
+      });
+    `,
+    'b.spec.js': `
+      const { test } = pwt;
+      test('passed short', async () => {
+        await new Promise(f => setTimeout(f, 100));
+      });
+      test('interrupted counts as skipped', async () => {
+        console.log('\\n%%interrupted');
+        await new Promise(f => setTimeout(f, 10000));
```

Comment:
can we make these a bit smaller numbers. I'd really like to avoid a 10 second wait here